components/LandingPage.tsx

```diff
@@ -199,12 +199,12 @@ const LandingPage: React.FC = () => {
           Get Started
         </a>
       </div>
-      <footer className="mt-5 font-weight-light text-center mb-5 text-secondary">
+      <footer className="mt-5 font-weight-light text-center pb-5 text-secondary">
```

Comment:
did you mean to move these into the `footer` component?

Reply:
What's the purpose of changing margin to padding?

Reply:
colors I'd imagine. If you do a padding the footer takes the footer color, so I think the best practice is to use padding.

Reply:
Yes it was because of the background